hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
Please avoid taking unrelated changes.  You did not branch from `hw1`.

Reply:
ok got it! thank you, professor!